test/modules/lydia/modSameNameAsFileBadRequire.chpl

```diff
@@ -0,0 +1,7 @@
+require "modSameNameAsFileGood.chpl";
```

Comment:
Can you make a test that triggers the warning whilst having both a top-level `require` and `use` ?